friends/src/components/Friends.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+class Friends extends React.Component {
+    constructor(){
+        super();
+        this.state = { 
+            friends: [],
+            error: ''
+        };
+    }
+
+     componentDidMount(){
+         console.log('cdm running');
+
+        axios
+            .get('http://localhost:5000/friends')
+            .then(res => {
+                console.log(res);
+                this.setState({ friends: res.data });
+              })
+              .catch(err => {
+                console.log(err);
+                this.setState({ error: err });
+              });
+     }
```

Comment:
Nice work! You'll use this syntax a whole lot, it'll be important to remember and may be worth writing down until you can memorize it